### PR TITLE
chore: Replace node-sass with sass (#125)

### PR DESCRIPTION
**Pull request by izkmdz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
   },
   "homepage": "https://github.com/silvermine/videojs-chromecast#readme",
   "dependencies": {
+    "sass": "^1.52.2",
```
Comment (onebytegone) [inline package.json:37]: Just checking, should this be a dev dependency? Also:

```suggestion
    "sass": "1.52.2",
```

**Review by onebytegone — CHANGES_REQUESTED** [review]

File: Gruntfile.js [hunk 2]
```diff
@@ -7,7 +7,7 @@
 var path = require('path'),
     getCodeVersion = require('silvermine-serverless-utils/src/get-code-version'),
     join = path.join.bind(path),
-    nodeSass = require('node-sass');
+    nodeSass = require('sass');
```
Comment (onebytegone) [inline Gruntfile.js:10]: Sorry, just saw this when submitting the review. We probably should also rename `nodeSass` since it's not node-sass anymore.

**Comment by izkmdz** [comment]
Thank you @onebytegone for the review! Pushed some minor changes, ready for re-review. 👍

File: package.json (hunk 1 above)
Comment (onebytegone) [reply package.json:37]: Is this still supposed to be a dependency?

**Review by onebytegone — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (izkmdz) [reply package.json]: Sorry about that. Did not notice it was not removed as a dependency when reinstalling with `--save-dev`. Should be good to go now.

**Review by onebytegone — APPROVED** [review]

**Comment by onebytegone** [comment]
Thanks @izkmdz!

**Pull request merged**